lib/spack/spack/ci.py

```diff
@@ -675,6 +685,13 @@ def job_query(job):
         return self.ir
 
 
+#: YAML fragment for retries
+_service_job_retries = {
```

Comment:
To me it looks like this is only used in one place, for rebuild index retries.  Could it just go inline there instead of coming up to the module level?

Reply:
good call, this was written before the noop job was stripped down.